AllReadyApp/Web-App/AllReady/Views/Manage/Index.cshtml

```diff
@@ -65,7 +82,8 @@
     @using (Html.BeginForm())
     {
         @Html.AntiForgeryToken()
-
+        <input asp-for="IsProfileComplete" type="hidden"/>
+        <input asp-for="ProfileCompletenessWarnings" type="hidden" />

```

Comment:
You are correct. They were needed originally when I was doing this differently. I will fix